eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/PointerStatements.scala

```diff
@@ -182,6 +220,17 @@ object PointerStatements {
 
   object SelectTopics {
 
+    def apply[F[_]: Monad](
+      legacy: SelectTopics[F],
```

Comment:
could we abstract away from `legacy` & `select `, I'd better use `first` / `second` ?

Reply:
done